instrumentation/apache-httpclient/apache-httpclient-5.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apachehttpclient/v5_0/ApacheHttpClientInstrumentationModule.java

```diff
@@ -21,6 +20,7 @@ public ApacheHttpClientInstrumentationModule() {
 
   @Override
   public List<TypeInstrumentation> typeInstrumentations() {
-    return singletonList(new ApacheHttpClientInstrumentation());
+    return List.of(
+        new ApacheHttpClientInstrumentation(), new ApacheHttpAsyncClientInstrumentation());
```

Comment:
The javaagent is compiled against Java 8, unfortunately you can't use Java 9+ methods/features:
```suggestion
    return Arrays.asList(
        new ApacheHttpClientInstrumentation(), new ApacheHttpAsyncClientInstrumentation());
```